lib/measured/base.rb

```diff
@@ -6,10 +6,19 @@ module Measured
   class UnitError < StandardError ; end
 
   class << self
+    def build(&blk)
+      builder = ConversionBuilder.new
+      builder.instance_eval(&blk)
+
+      clazz = Class.new(Measurable)
+      clazz.instance_variable_set(:@conversion, builder.conversion)
+      clazz
```

Comment:
This might be a little easier on the eyes (also less code):

```ruby
      Class.new(Measurable) do
        @conversion = builder.conversion
      end
```

Not sure though, what do you think?